features/src/rapids-build-utils/devcontainer-feature.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "NVIDIA RAPIDS devcontainer build utilities",
   "id": "rapids-build-utils",
-  "version": "25.2.6",
+  "version": "25.2.8",
```

Comment:
Bumping by 2 versions in anticipation of https://github.com/rapidsai/devcontainers/pull/440 merging first.

TODO before merging: confirm this version bump is still right after merging in latest `branch-25.02`.

Reply:
#440 is merged, yes this is the right change